packages/core-utils/src/map.js

```diff
@@ -106,11 +106,13 @@ export function itineraryToTransitive(
       leg.mode === "WALK" ||
       leg.mode === "BICYCLE" ||
       leg.mode === "CAR" ||
-      leg.mode === "MICROMOBILITY"
+      leg.mode === "MICROMOBILITY" ||
```

Comment:
We could probably use `isAccessMode` for the entire `if` condition.

Reply:
Good call thanks! Refactored in e351b70d8907551a03043bf49d583317231644d2